roles/openshift_metrics/defaults/main.yaml

```diff
@@ -4,8 +4,14 @@ image_version: latest
 master_url: https://kubernetes.default.svc.cluster.local
 
 hawkular_user_write_access: False   #if user accounts should be able to write metrics
+hawkular_cassandra_nodes: 1
+# emptydir|pv|dynamic
+hawkular_cassandra_storage_type: emptydir
+hawkular_cassandra_pv_prefix: metrics-cassandra
+hawkular_cassandra_pv_size: 10Gi
 
 heapster_standalone: False
+heapster_allowed_users: system:master-proxy
```

Comment:
My thought is that if its pertinent to the role or running it in the playbook, it should be described here. I'm sure we'll have some overlap for a while between here and origin-metrics and as the use of the deployer drops off and this role is used more for installing we'll start to see things removed from origin-metrics.

I'm not sure the policy on describing roles in the docs, but that would make sense as well. At a minimum I'd feel they would be described in documentation for the enterprise (and I suppose origin too) community, in some capacity.